presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/BenchmarkReorderJoins.java

```diff
@@ -0,0 +1,118 @@
+/*
```

Comment:
yes agreed, it would help performance for most cases.  I meant we shouldn't change the limit.